src/main/resources/config/application.yml

```diff
@@ -80,7 +80,7 @@ eureka:
             version: ${info.project.version}
     client:
         enabled: true
-        fetch-registry: false
```

Comment:
Doesn't this line cause a stack trace on boot since the registry will initially fail to connect to itself ? But anyway I agree that we should have this.

Reply:
Yes there is a stacktrace on startup but works otherwise:
```
2018-06-30 13:37:02.315  WARN 45706 --- [           main] c.n.d.s.t.d.RetryableEurekaHttpClient    : Request execution failed with message: java.net.ConnectException: Connection refused (Connection refused)
2018-06-30 13:37:02.316 ERROR 45706 --- [           main] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_JHIPSTER-REGISTRY/jhipsterRegistry:0993f0bd07ba4c1f2895eb5d238a5735 - was unable to refresh its cache! status = Cannot execute request on any known server
```

Reply:
I guess there is no way around it, especially if we want to do this: https://github.com/jhipster/generator-jhipster/issues/7496 then the registry must be itself available in the list of available instances.

Reply:
@jdubois do you know a way to hide this stacktrace, maybe a logback config ?

Reply:
Well, I kind of like it as it shows the error -> we could have a look at wrapping this into an ERROR message that would be clearer, and hide the stacktrace